src/appservice/AppService.ts

```diff
@@ -226,11 +231,22 @@ export class MjolnirAppService {
     Logger.configure(config.logging ?? { console: "debug" });
     const dataStore = new PgDataStore(config.db.connectionString);
     await dataStore.init();
+    const storagePath = path.isAbsolute(config.dataPath)
+      ? config.dataPath
+      : path.join(__dirname, "../", config.dataPath);
+    const eventDecoder = DefaultEventDecoder;
+    const backingStore = config.roomStateBackingStore.enabled
+      ? new SqliteRoomStateBackingStore(
+          path.join(storagePath, "room-state-backing-store.db"),
+          eventDecoder
+        )
+      : undefined;
```

Comment:
Do you think it's worth making the path handling (and creating the store) consistent with src/index.ts somehow? like a utility function in `SqliteRoomStateBackingStore`?

Reply:
Pass the event decoder as an argument to this function though

Reply:
/me will check/do that yup